tests/ProtoTests/BasicPaginated/Gapic/BasicPaginatedGapicClient.php

```diff
@@ -0,0 +1,231 @@
+     *           will be ignored.
+     *     @type array $credentialsConfig
+     *           Options used to configure credentials, including auth token caching, for the
+     *           client. For a full list of supporting configuration options, see
+     *           {@see CredentialsWrapper::build()}
+     *     @type bool $disableRetries
+     *           Determines whether or not retries defined by the client configuration should be
+     *           disabled. Defaults to `false`.
+     *     @type string|array $clientConfig
+     *           Client method configuration, including retry settings. This option can be either
+     *           a path to a JSON file, or a PHP array containing the decoded JSON data. By
+     *           default this settings points to the default client config file, which is
+     *           provided in the resources folder.
+     *     @type string|TransportInterface $transport
+     *           The transport used for executing network requests. May be either the string
+     *           `rest` or `grpc`. Defaults to `grpc` if gRPC support is detected on the system.
+     *           *Advanced usage*: Additionally, it is possible to pass in an already
+     *           instantiated {@see TransportInterface} object. Note that when this object is
+     *           provided, any settings in `$transportConfig`, and any `$apiEndpoint` setting,
+     *           will be ignored.
+     *     @type array $transportConfig
+     *           Configuration options that will be used to construct the transport. Options for
+     *           each supported transport type should be passed in a key for that transport. For
+     *           example:
+     *           ```
+     *           $transportConfig = [
+     *               'grpc' => [
+     *                   '...' => '...',
+     *               ],
+     *               'rest' => [
+     *                   '...' => '...',
+     *               ],
+     *           ];
+     *           ```
+     *           See the {@see GrpcTransport::build()} and {@see RestTransport::build()} methods
+     *           for the supported options.
+     * }
+     *
+     * @throws ValidationException
+     *
+     * @experimental
+     */
+    public function __construct(array $options = [])
+    {
+        $clientOptions = $this->buildClientOptions($options);
+        $this->setClientOptions($clientOptions);
+    }
+
+    /**
+     *
+     * Sample code:
+     * ```
+     * $basicPaginatedServiceClient = new BasicPaginatedClient();
+     * try {
+     *     // Iterate over pages of elements
+     *     $pagedresponse = $basicPaginatedServiceClient->methodPaginated();
+     *     foreach ($pagedresponse->iteratePages() as $page) {
+     *         foreach ($page as $element) {
+     *             // doSomethingWith($element);
+     *         }
+     *
+     *     }
+     * // Alternatively:
```

Comment:
I assume this is the same whitespace issue that you already had?

Reply:
Indeed yes. To be fixed later.